common/genes/05_genes_special_accessories_clothes.txt

```diff
@@ -1361,6 +1361,18 @@ special_genes = {
 				boy = male
```

Comment:
Could you resave the crown textures as well? It has no alpha too,
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/122634133-8b6dfd80-d0ed-11eb-81ec-93c3bbb810b0.png)

</details>

Reply:
done